src/deploy.py

```diff
@@ -129,6 +129,14 @@ def configure_montagu(service, data_exists):
     configure_api(service, passwords['api'], token_keypair_paths,
                   service.settings["hostname"], is_prod,
                   service.settings["orderly_web_api_url"])
+
+    task_queue_user = "MONTAGU_TASK_QUEUE" if service.settings["use_real_diagnostic_reports"] else "test.user@example.com"
+    # TODO: Add the user and password if does not exist and grant required perms (run, publish)
```

Comment:
Is there a ticket for this, or were you planning on doing it here?

Reply:
There's a ticket: https://mrc-ide.myjetbrains.com/youtrack/issue/VIMC-4218